### DISCUSSION TITLE
[Stundenplan, Mensaplan, Informationssystem] Übersichtsseite Schüler/Lehrer

### DISCUSSION THREAD

**Discussion by Schmarvinius** [SE-TINF22B2/G2-Schoolify#122] [User Stories]
### User Story
Als Schüler oder Lehrer möchte ich eine Übersichtsseite haben, auf der die wichtigsten Sachen stehen, um alle wichtigen Aspekte für meinen Schultag auf einer Seite zu sehen.
### Beschreibung
Nach dem Einloggen auf die Seite möchte ich auf eine Startseite gelangen auf der mein heutiger Stundenplan und Mensaplan steht, dazu sollen News und anstehende Termine angezeigt werden.

### Akzeptanz-Kriterien
- [ ] Stundenplan wird angezeigt
- [ ] Mensaplan wird angezeigt
- [ ] Aktuelle News werden angezeigt
- [ ] Anstehende Termine werden angezeigt

**Comment by CodeRider101** [comment]
Inhalt der Übersichtsseite erstmal auf aktuelle Nachrichten beschränkt.
